report/2.Analysis/ProblemStatement.tex

```diff
@@ -81,30 +81,30 @@ \subsection{Requirements}\label{subsec:requirements}
 Finally, the \textit{won't have} requirements are deemed outside the scope of the current project and focus on the future application of the device, which would require more efficient projectiles in order to take down the observed target and being able to differentiate between whether the object is a threat or not.
 
 \subsection{Test cases}
-To determine if the requirements have been fulfilled, in a completely binary sense, we created a couple of test cases, or scenarios, which the solution should be able to complete.
+To determine if the requirements have been fulfilled, in a completely binary sense, a couple of test cases, or scenarios in which the solution should be able to complete was made.
 This subsection will present four test cases, two \textit{should have} and two \textit{could have}, corresponding with the MoSCoW naming, and argue why these are realistic and reflect the requirements.
 When doing these tests, the target should be a color that contrasts the surrounding area.
 As our group rooms primarily have white walls, a good option is red.
-Some test-cases will mention a balloon, which is 27 centimetres in diameter, while others mention a rubber ball, which is approximately 5-10 centimetres in diameter.
+Some test-cases will mention a balloon, which is 27 centimeters in diameter, while others mention a rubber ball, which is approximately 5-10 centimeters in diameter.
 All distances between the turret and the target are only based on the horizontal plane, which means the vertical distance is excluded.
 
-\subsubsection{Dropping a ballon}
+\subsubsection{Dropping a balloon}
 \textit{A balloon is dropped from the ceiling, approximately 2-2.5 meters from the floor.
 The \texttt{F.L.A.T} should hit the balloon before it hits the ground.
-The \texttt{F.L.A.T} should be approximately 4 meters away from the ballon.}
+The \texttt{F.L.A.T} should be approximately 4 meters away from the balloon.}
 
 It can be difficult to fully predict the speed at which the device will move, due to lacking knowledge about the motors, and due to no realistic way to assume the execution speed of the different software parts.
 This example has some time restrictions, as the balloon will need to be hit before it hits the ground, however it should still be accomplishable.
 
 \subsubsection{Rolling a ball}
-\textit{The \texttt{F.L.A.T} sits on the edge of a table, approximately 80 centimetres tall.
+\textit{The \texttt{F.L.A.T} sits on the edge of a table, approximately 80 centimeters tall.
```

Comment:
Suggestion: \textit{The \texttt{F.L.A.T} sits on the edge of a table, that is approximately 80 centimeters tall.